packages/block-library/src/image/edit.js

```diff
@@ -497,188 +511,196 @@ export class ImageEdit extends Component {
 			</>
 		);
 
+		const AlignmentWrapper = needsAlignmentWrapper ? Block.div : Fragment;
```

Comment:
Would it make any sense to always have an alignment wrapper?

Reply:
Even if we do add it always, we still have specific behavior: If the wrapper is "Fragment" it means the figure needs to become Block.figure.

Reply:
Ah, I see it's similar on the front-end. Sounds good then. :)